### PR TITLE
new version output

### PR DESCRIPTION
**Pull request by C0DK**
I've changed the output of `jrnl -v` as mentioned in https://github.com/jrnl-org/jrnl/pull/679 - I haven't modified the tests, as i want to make sure you are okay with the representation before hand. 

### Checklist
- [x] The code change is tested and works locally.
- [ ] Tests pass. Your PR cannot be merged unless tests pass
- [x] There is no commented out code in this PR.
- [x] Have you followed the guidelines in our Contributing document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?

### CODE REVIEW

File: jrnl/cli.py [hunk 1]
```diff
@@ -140,7 +140,7 @@ def run(manual_args=None):
     configure_logger(args.debug)
     args.text = [p.decode('utf-8') if util.PY2 and not isinstance(p, unicode) else p for p in args.text]
     if args.version:
-        version_str = "{0} version {1}".format(jrnl.__title__, jrnl.__version__)
+        version_str = "{0} version {1} (Python {3})".format(jrnl.__title__, jrnl.__version__, sys.version)
```
Comment (peter-schmidbauer) [inline jrnl/cli.py]: This throws an error currently: 

```suggestion
        version_str = "{0} version {1} (Python {2})".format(jrnl.__title__, jrnl.__version__, sys.version)
```

**Comment by wren** [comment]
This is what outputs for me:
```
jrnl version 2.0.0 (Python 3.7.4 (default, Oct  5 2019, 11:45:34)
[Clang 10.0.1 (clang-1001.0.46.4)])
```

The use of `sys.version` seems like overkill. How about using `platform.python_version` or `sys.version_info` to get a better formatted output?

Thanks for adding this!

**Comment by C0DK** [comment]
We can close this as #719 is a better bet

**Comment by wren** [comment]
@C0DK This is looking good. Since you were working on it first, I'd prefer to have you finish it. Do you still have time for it?

**Comment by C0DK** [comment]
@wren  I can change it to `sys.version_info` and then create a unit test, now, sure :) - But it isn't that important for me to get credit hehe. So whether it's my PR or his, is the same to me.

**Comment by wren** [comment]
@C0DK If you have the time, I'd prefer for you to do it. Thank you!

**Pull request closed without merging**